### PR TITLE
[BUGFIX] Fix issue with LogMessageFatal

### PR DESCRIPTION
**Pull request by anko-intel**
## Description ##
There were two different versions of the LogMessageFatal class in the
same namespace in different files so depending on order of compilation
it sometimes cause mixing it and segmentation fault (probably due to
mixing versions of constructor/destructor (one inlined and one linked))
It is connected with the issue reported on v1.x branch: https://github.com/apache/incubator-mxnet/issues/20827

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### CODE REVIEW

**Review by bartekkuncer — APPROVED** [review]

File: 3rdparty/mshadow/mshadow/base.h [hunk 1]
```diff
@@ -314,7 +314,7 @@ MSHADOW_HALF_BF_OPERATOR(bool, >=)
 /*! \brief overloaded <= operator between half_t and bf16_t */
 MSHADOW_HALF_BF_OPERATOR(bool, <=)
 
-#include "./logging.h"
```
Comment (mozga-intel) [inline 3rdparty/mshadow/mshadow/base.h:317]: @anko-intel: Why are you modifying these files here?

**Review by mozga-intel — COMMENTED** [review]
@anko-intel: Why are these files modified here?

**Comment by anko-intel** [comment]
> @anko-intel: Why are these files modified here?

3rdparty/mshadow is no git module any longer.  The other words it is not real 3rparty now

**Comment by anko-intel** [comment]
@mxnet-bot run ci [centos-gpu, centos-cpu]

**Review by josephevans — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**